util/util_test.go

```diff
@@ -0,0 +1,20 @@
+package util
+
+import (
+	"reflect"
+	"testing"
+)
+
+func TestFindBasesFromVendor(t *testing.T) {
+	vendorPath := "../fixtures/vendor/"
```

Comment:
I suggest using this approach to test fixtures: https://dave.cheney.net/2016/05/10/test-fixtures-in-go